src/HybridSystems.jl

```diff
@@ -29,18 +60,16 @@ A hybrid system modelled as a hybrid automaton.
 - `switchings` -- vector of switchings
 - `ext`        -- dictionary that can be used by extensions
 """
-struct HybridSystem{A, S, I, G, R, W} <: AbstractHybridSystem
+struct HybridSystem{A, S, R, W} <: AbstractHybridSystem
```

Comment:
`invariants` and `guards` can be removed from the docstring too.

Reply:
Woops :-P